modules/colossussspBidAdapter.js

```diff
@@ -3,8 +3,8 @@ import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes';
 import * as utils from '../src/utils';
 
 const BIDDER_CODE = 'colossusssp';
-const URL = '//colossusssp.com/?c=o&m=multi';
-const URL_SYNC = '//colossusssp.com/?c=o&m=cookie';
+const G_URL = '//colossusssp.com/?c=o&m=multi';
+const G_URL_SYNC = '//colossusssp.com/?c=o&m=cookie';
```

Comment:
Variable names were changed due to collision with global javascript object name "URL"

Reply:
Prefix G_ jut means "global"

Reply:
About 404 code for https://supply.colossusssp.com/img/logo.png. We have been using our log as test creative, but link have changed. I fixed those link

Reply:
Hi @HuddledMasses 

Thanks for the clarification on the variable name and the image URL.

Are you going to also change the protocol for these URLs to ensure they're `https` for 3.0?

Reply:
For that moment we will leave http possible, due to limited hardware resource. After we scale in close future we will switch to strict https

Reply:
@HuddledMasses 

Thanks for confirming.  For Prebid 3.0 there is a hard-requirement that all adapters will need to use `https` for their external requests.  We were planning to manually update all adapter's endpoints to `https` before the change if they weren't already converted.

However if your endpoint doesn't support the capacity yet, we'll have to temporarily drop the adapter as part of the 3.0 release.  Once your endpoint has the capacity and you can handle the permanent `https` switch, you can resubmit the adapter to get it back into the 3.x versions of Prebid.

CC @bretg @mkendall07 to comment further (if needed).

I'll recheck the PR as is now to make sure it's good merge (since we're still in the 2.x versions of Prebid production-wise) so we can get the other changes put into place.

Reply:
I've added strict https, to leave adapter in future major release. In a week we will optimize our
 HW resources to handle more https traffic

Reply:
@HuddledMasses Thank you making the update.  It seems there are two test cases that need to be updated in lieu of the protocol change.  They're at lines 44 & 119 in the spec file.  Can you make those changes please?

Reply:
Oh yes, I've added them

Reply:
@mkendall07 and I agree that HTTPS is a hard requirement for 3.0.

Reply:
@bretg We made https strict earlier in this pool request